Examples/Sources/MacroExamples/Implementation/Expression/FontLiteralMacro.swift

```diff
@@ -40,6 +40,9 @@ private func replaceFirstLabel(
   }
 
   var tuple = tuple
-  tuple[tuple.startIndex] = firstElement.with(\.label, .identifier(newLabel))
+  tuple[tuple.startIndex] =
+    firstElement
+    .with(\.label, .identifier(newLabel))
+    .with(\.colon, .colonToken())
```

Comment:
My preferred way of spelling this is to actually modify the node instead of using `with`. In my experience it shortens the code and makes it easier to read.

```swift
tuple[tuple.startIndex].label = .identifier(newLabel)
tuple[tuple.startIndex].colon = .colonToken()
```

And yes, I know that the current examples use `with` everywhere. If you’d be interested in migrating many of those cases to variable setters, that would be amazing. I wouldn’t make it part of this PR though.

Reply:
Sure thing. I've added an issue to track all enhancements to macro examples including this remake: #2169. Thx.